Becoming a developer (#684)
ember-learn/ember-blog: source/2020-06-19-the-ember-times-issue-153.md

---

## [Stories of becoming a developer 📖](https://dev.to/agathebadia/share-your-dev-story-51dg)

> [Agathe Badia (@honeypotio)](https://github.com/Agathebadia) has written a blog post called **Share your Dev story**, this blog post introduces her new website 🎉 that was built with [ember.js](https://emberjs.com/), and it is called [Becoming a Developer](https://becoming-a-developer.netlify.app/). 

Reviewer: ```suggestion
[Agathe Badia (@honeypotio)](https://github.com/Agathebadia) has written a blog post called **Share your Dev story**. This blog post introduces her new website 🎉 that was built with [ember.js](https://emberjs.com/). It is called [Becoming a Developer](https://becoming-a-developer.netlify.app/). 
```

> This site is where developers can share their background, the story of how they started, interest in development, studies, first developer role, and some recommendations for anyone starting or continuing to dive into development. 💻

Reviewer: I think it may be nice to change the article to "you" here, so that a reader may be encouraged to write a post on Agathe's website. What do you think?

```suggestion
On Becoming a Developer, you can share your background, the story of how you started, interest in coding, studies, first developer role, and some recommendations for anyone starting or continuing to dive into development. 💻
```

> In her [Share your Dev story](https://dev.to/agathebadia/share-your-dev-story-51dg) post, she talks about how meaningful her new website is to her, and her passion as a Jr. developer shines. 🌟

Reviewer: I think the "and" felt a bit awkward. Maybe break into two sentences?
```suggestion
In her [Share your Dev story](https://dev.to/agathebadia/share-your-dev-story-51d) post, Agathe talks about how meaningful her new website is to her. You can see her passion as a developer shine. 🌟
```

> The Developers' stories shared are **inspiring** and **insightful**, and developers are telling their stories with all kinds of different backgrounds. Check it out, and it could inspire **you** to do something you never thought possible! 😃

Reviewer: I think we could rearrange the parts for a stronger conclusion. What do you think about:
```suggestion
Each developer's story is **unique** and **insightful** and can inspire _you_ to do something you never thought possible. [Check it out](https://becoming-a-developer.netlify.app/) and be sure to share your own! 😃
```
?

## [Stories of becoming a developer 📖](https://dev.to/agathebadia/share-your-dev-story-51dg)

> [Agathe Badia (@honeypotio)](https://github.com/Agathebadia) has written a blog post called **Share your Dev story**. This blog post introduces her new website 🎉 that was built with [ember.js](https://emberjs.com/). It is called [Becoming a Developer](https://becoming-a-developer.netlify.app/). 

Reviewer: ```suggestion
[Agathe Badia (@honeypotio)](https://github.com/Agathebadia) has written a blog post called **Share your Dev story**. This blog post introduces her new website 🎉 that was built with [Ember.js](https://emberjs.com/). It is called [Becoming a Developer](https://becoming-a-developer.netlify.app/). 
```

On Becoming a Developer, you can share your background, the story of how you started, interest in coding, studies, first developer role, and some recommendations for anyone starting or continuing to dive into development. 💻

> In her [Share your Dev story](https://dev.to/agathebadia/share-your-dev-story-51d) post, Agathe talks about how meaningful her new website is to her. You can see her passion as a developer shine. 🌟

Reviewer: ```suggestion
In her [Share your Dev story](https://dev.to/agathebadia/share-your-dev-story-51dg) post, Agathe talks about how meaningful her new website is to her. You can see her passion as a developer shine. 🌟
```
